src/cli/main.rs

```diff
@@ -0,0 +1,84 @@
+use clap::{arg, value_parser, App, Arg, SubCommand};
+use std::{collections::HashMap, fmt, fs::File, io::Read, path::PathBuf};
+
+use specs::types::Value;
+use wasmi::{ImportsBuilder, NopExternals};
+
+use delphinus_zkwasm::cli::run;
+
+/// Simple program to greet a person
+fn main() {
+    let wasm_file_arg = Arg::with_name("wasm_file")
+        .short('w')
+        .long("wasm")
+        .value_name("FILEPATH")
+        .help("Path of the wasm file")
+        .required(true)
+        .takes_value(true)
+        .value_parser(value_parser!(std::string::String));
+    let fn_name_arg = Arg::with_name("function_name")
+        .long("fname")
+        .short('f')
+        .required(true)
+        .value_name("FUNCTIONNAME")
+        .help("Function you would like to run from the file")
+        .takes_value(true)
+        .value_parser(value_parser!(std::string::String));
+    let type_arg = Arg::with_name("types")
```

Comment:
We can fetch types from interpreter, for example,
```
        if let Some(ExternVal::Func(func)) = compile_outcome.instance.export_by_name("test") {
            println!("{:?}", func.signature());
        }
```

Reply:
removed type argument.